src/text.jl

```diff
@@ -83,10 +83,22 @@ immutable GtkTextRange <: Range
     GtkTextRange(a,b) = new(mutable(copy(a)),mutable(copy(b)))
 end
 
-#type GtkClipboard
-#TODO
-#end
-
+#####  GtkClipboard #####
+
+GtkClipboardLeaf(selection::UInt16) =  GtkClipboardLeaf(ccall((:gtk_clipboard_get,libgtk), Ptr{GObject},
+    (UInt16,), selection))
+GtkClipboardLeaf() = GtkClipboardLeaf(Gtk.GdkAtoms.CLIPBOARD)
+clipboard_set_text(clip::GtkClipboard,text::String) = ccall((:gtk_clipboard_set_text,libgtk), Void,
+    (Ptr{GObject}, Ptr{Uint8},Cint), clip, text, length(text))
```

Comment:
I believe this will be exposed automatically as `G_.text(clipboard, text)` once the autogen script is rerun.